opentelemetry-sdk/tests/metrics/test_metrics.py

```diff
@@ -126,13 +140,16 @@ def test_record_batch_exists(self):
         self.assertEqual(handle.aggregator.current, 2.0)
 
     def test_create_metric(self):
-        meter = metrics.MeterProvider().get_meter(__name__)
+        resource = mock.Mock(spec=resources.Resource)
```

Comment:
is there a need to create a mock here, since there is a Resource object in the SDK that works?

Reply:
I don't have a strong opinion there. I updated to use the real object.